.github/workflows/reusable-dev-deployment.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           env: ${{ inputs.name }}
 
       - name: Comment progress on PR
-        uses: thollander/actions-comment-pull-request@v2
+        uses: thollander/actions-comment-pull-request@v3
         if: ${{ inputs.env == 'feature-branch' }}
         with:
           pr_number: ${{ inputs.pr_number }}
```

Comment:
according to https://github.com/thollander/actions-comment-pull-request/releases/tag/v3.0.0
we should rename these parameters:

- From pr_number to pr-number
- From comment_tag to comment-tag
- From create_if_not_exists to create-if-not-exists